src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddFileMetadataToDepositSpec.scala

```diff
@@ -160,24 +219,25 @@ class AddFileMetadataToDepositSpec extends UnitSpec with BeforeAndAfter {
       case Failure(ActionException(_, message, _)) =>
         message should {
           include("Values found for these columns: [SF_COLLECTION]") and
-            include("these columns should be empty because there are no audio/video")
+            include("these columns should be empty because there are no audio/video files found in this dataset")
```

Comment:
I'm not sure if you're aware, but the test starting at 202 ends at 209, while 219 is part of the test that starts at 211 and ends at 225